specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json

```diff
@@ -1,132 +1,11996 @@
+            "description": "The requested API version",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Job parameters supplied by user.",
+            "schema": {
+              "$ref": "#/definitions/SatelliteIngestionJobRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SatelliteIngestionJobResponse"
+            }
+          },
+          "default": {
+            "description": "Error"
+          }
+        },
+        "x-ms-examples": {
+          "Scenes_CreateSatelliteDataIngestionJob": {
+            "$ref": "./examples/Scenes_CreateSatelliteDataIngestionJob.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Scenes"
+        ],
+        "description": "Get satellite data ingestion job's details.",
+        "operationId": "Scenes_GetSatelliteDataIngestionJobDetails",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "jobId",
+            "description": "Id of the job.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The requested API version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SatelliteIngestionJobDetails"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Scenes_GetSatelliteDataIngestionJobDetails": {
+            "$ref": "./examples/Scenes_GetSatelliteDataIngestionJobDetails.json"
+          }
+        }
+      }
+    },
+    "/scenes/downloadFiles": {
+      "get": {
+        "tags": [
+          "Scenes"
+        ],
+        "description": "Downloads and returns file content as response for the given input filePath.",
+        "operationId": "Scenes_Download",
+        "produces": [
+          "application/octet-stream",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "filePath",
+            "description": "cloud storage path of scene file.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The requested API version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "format": "byte",
+              "type": "string"
```

Comment:
This probably doesn't have Vivek's changes. The type should be `file`.

Reply:
Fixed